### PR TITLE
chore: publishing npm package on tagged builds

### PR DESCRIPTION
**Pull request by markmcdowell**
A package will be built and published to npm.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@reactivemarkets/platform-api",
+  "version": "0.0.0",
+  "description": "Generated api for Reactive Markets Platform",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/js/Message_generated.js",
```
Comment (markaylett) [inline package.json:8]: Tags will be taken from the "latest" branch. The generated code on the latest branch will be in `js`, not `build/js`.

Comment (markmcdowell) [reply package.json]: Fixed.

**Comment by markmcdowell** [comment]
@markaylett oh actually i've setup the github workflow on tags so it wouldn't be on the latest branch.

**Comment by markaylett** [comment]
> @markaylett oh actually i've setup the github workflow on tags so it wouldn't be on the latest branch.

That's fine. We just tag the latest branch.

**Pull request merged**